widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Base.kt

```diff
@@ -12,6 +12,10 @@ import com.glia.widgets.view.unifiedui.theme.base.*
 internal fun DefaultTheme(pallet: ColorPallet?): UnifiedTheme? = pallet?.let {
     UnifiedTheme(
         callTheme = CallDefaultTheme(it),
-        chatTheme = ChatDefaultTheme(it)
+        chatTheme = ChatDefaultTheme(it),
+        alertTheme = AlertDefaultTheme(it),
+        bubbleTheme = null,
+        surveyTheme = null,
+        callVisualizerTheme = null
```

Comment:
These are not going to be filled? I thought those would use the classes that you added. E.g: BubbleDefaultTheme

Reply:
will be fully filled on future PRs